addons/mail/static/src/messaging/component/chatter_topbar/chatter_topbar.xml

```diff
@@ -26,31 +26,29 @@
                 </t>
                 <t t-if="chatter.hasActivities">
                     <button class="btn btn-link o_ChatterTopbar_buttonScheduleActivity" type="button" t-att-disabled="chatter.isDisabled" t-on-click="_onClickScheduleActivity">
-                        <i class="fa fa-clock-o"/>
+                        <i class="fa fa-clock-o" aria-hidden="true"/>
                         Schedule activity
                     </button>
                 </t>
                 <button class="btn btn-link o_ChatterTopbar_buttonAttachments" type="button" t-att-disabled="chatter.isDisabled" t-on-click="_onClickAttachments">
-                    <i class="fa fa-paperclip"/>
+                    <i class="fa fa-paperclip" aria-hidden="true"/>
                     <t t-if="chatter.isDisabled or chatter.thread and chatter.thread.areAttachmentsLoaded">
                         <span class="o_ChatterTopbar_buttonCount o_ChatterTopbar_buttonAttachmentsCount" t-esc="chatter.thread ? chatter.thread.allAttachments.length : 0"/>
                     </t>
                     <t t-else="">
-                        <i class="o_ChatterTopbar_buttonAttachmentsCountLoader fa fa-spinner fa-spin"/>
+                        <i class="o_ChatterTopbar_buttonAttachmentsCountLoader fa fa-spinner fa-spin" aria-label="Attachment counter loading..."/>
```

Comment:
title

Reply:
Idea was icon with `fa` was expressive enough without aria

Reply:
I don't think it ever hurts to add a title instead but fine either way.

Reply:
Note that all of the icons that would be considered "not expressive enough" where title would be justified are therefore bad icons.

Reply:
Yeah, it's a big topic in UX/UI world: https://ux.stackexchange.com/a/1803
That's the same idea but here with title: somewhat better (label doesn't take permanent space visually) and worse (blocks part of UI on hover, cursor position annoyance, no a solution in environment without a cursor like touch-screen, etc.).